mantidproject/mantid
# Issue and Pull Request templates

The work is to change the issue and PR templates to try to ensure that feedback is given to the original problem reporter. Eg if an instrument scientist reports a bug to developer, the developer raises the issue, but when the final PR is merged the instrument scientist should be notified.

ISSUE and PULL_REQUEST templates modified

**To test:**

Make sure that the new templates make sense.

Is it clear that the person must be notified when the issue is closed.

**Release Notes** 

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
